### PR TITLE
Import from Solid Authentication Panel repository

### PR DESCRIPTION
**Pull request by acoburn**
This imports the Solid-OIDC text and associated artifacts from the solid/authentication-panel in their current state.

We may want to adjust the general layout -- the specification is in the base path and the primer is in a `./primer` directory. I'm not committed to that structure.

I made the appropriate adjustments to the CI/CD workflows, but otherwise, everything is a direct copy over. Once this is merged, we'll want to adjust some of the details (repo location, etc)

### CODE REVIEW

File: .github/workflows/ci-config.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Workflow for processing and publishing bikeshed proposals to github pages
+
+name: verify proposals
+on:
+  push:
+    # only concerned with changes in the main branch
+    branches: [ main ]
```
Comment (matthieubosquet) [inline .github/workflows/ci-config.yml:7]: Does the verify need to run on push as well?

**Review by matthieubosquet — APPROVED** [review]
Looks good per https://github.com/solid/solid-oidc/runs/3086031984

Are PRs to main correctly gated with the build action and required review in the repo's settings?
Are direct pushes to main disabled?

**Comment by acoburn** [comment]
> Are PRs to main correctly gated with the build action and required review in the repo's settings?

I don't have access to that level of this repository. Given that this build action doesn't currently exist in the repo, I can only imagine that the answer is no.

> Are direct pushes to main disabled?

Similar to the above. I haven't tried pushing directly to `main`. I suspect that direct pushes are not disabled

These are the sorts of repo setting details that will need to be sorted out, but I don't believe they need to block this current PR

cc @csarven

File: .github/workflows/ci-config.yml (hunk 1 above)
Comment (acoburn) [reply .github/workflows/ci-config.yml:7]: Probably not. That happens with the CD workflow. For the purposes of this PR, I really wanted to just copy what currently exists. We can modify/refine in subsequent PRs.

**Review by elf-pavlik — APPROVED** [review]

**Pull request merged**